source/_posts/2020-10-01-hacktoberfest-2020.markdown

```diff
@@ -0,0 +1,77 @@
+---
+title: "Hacktoberfest 2020"
+description: "It is October and Hacktoberfest started! Home Assistant participates again this year, and you?"
+date: 2020-10-01 00:00:00
+date_formatted: "October 1, 2020"
+author: Franck Nijhof
+author_twitter: frenck
+categories: Community
+og_image: /images/blog/2020-10-01-hacktoberfest-2020/social.png
+---
+
+<img src='/images/blog/2020-10-01-hacktoberfest-2020/social.png' style='border: 0;box-shadow: none;'>
+
+October has started, which means: [Hacktoberfest][hacktoberfest]!
+
+Hacktoberfest is a worldwide, month-long celebration of open source. An event
+open to everyone. Whether you’re a developer, student learning to code,
+documenter, designer, you can help drive the growth of open source projects,
+like Home Assistant.
+
+By participating in Hacktoberfest, and by contributing four GitHub pull
+requests, you will earn either a free t-shirt or, new this year, have a
+tree planted. All backgrounds and skill levels are encouraged to complete the
+Hacktoberfest challenge.
+
+Just like all other years, Home Assistant participates in Hacktoberfest.
+
+## What to work on?
+
+So, you’ve decided to earn yourself that free Hacktoberfest t-shirt? Great!
+We've collected a bunch of tasks/suggestions/bugs that we'd love to get
+some help on. You can find those by filtering/searching for issues with
+the [`hacktoberfest` label][all].
+
+- [Frontend Hacktoberfest issues][frontend]
```

Comment:
Can we highlight some easy to help issues for frontend/core. Martin has created a bunch of issues with their own label. Ie, help us rewrite tests from TestCase to PyTest.

Reply:
I kinda like the idea, and looked at it,
<https://github.com/home-assistant/core/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Aunittest.TestCase>

That would work, but for the others, it would be quite picking singles out of it (which probably end up being resolved quickly).